packages/e2e-test-utils/src/inserter.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
The rest of `@wordpress/e2e-test-utils` is architected as a one-function-per-file approach. This stands out as being inconsistent in how it exports many functions from the same file. Further, I've personally found some convenience in using VSCode Cmd+P file navigation using the name of an end-to-end test function, which doesn't work for things like `e2etestutilinsertblock`.

Reply:
yes, I talk about this here https://github.com/WordPress/gutenberg/pull/22414#discussion_r426518784  I personally struggle with all these files and find it to be very weird especially things that are related. I don't mind per file function but if we do so,  I'd prefer folders. 

In the future we might want to separate editor utils from frontend  utils from WP Admin utils... to be able to say: run tests on the playground... and having a single bucket, make this harder.

Reply:
I'd missed the earlier comment. I see arguments both ways to whether it should be architected this way in the first place. Ultimately though, the inconsistency is probably more concerning to me than one or the other approach used consistently.

>I personally struggle with all these files and find it to be very weird especially things that are related.

Could you elaborate on why this is a problem?

For me, the fact a folder contains a large number of files isn't inherently problematic. It's at least predictable in how a developer can discover the source (related to my original comment). I've seen that it's common to resist this way of structuring files toward some feeling of obligation toward fewer files, and I can't quite put my finger on why it is. But I often observe that it's this desire that can cause the creation of somewhat ad hoc categorization of a few things, rather than on the system as a whole. When this happens, I personally find it very difficult to navigate the code, because the organization is not immediately obvious.

What you mention as a separation between editor and front-end utilities is more appealing to me in regards to systematic categorization. I think it also feeds into thinking about the purpose of the package as a whole, and how the file structure relates to that. It may even raise the question if the package is focused enough in its goal, or if some of these specific needs should fall outside of the scope (e.g. their own packages).

Reply:
> Could you elaborate on why this is a problem?

I think it can be problem because you end up with several functions that do almost the same thing slightly differently and for me personally I know it discouraged me several times from adding new utilities. It felt that the utilities were too specific for a global  wp e2e test utils but reusable enough if we had a more specific context.